### PR TITLE
Improve the TTLRemover with last Cassandra 2.2

### PR DESCRIPTION
**Pull request by fasar**
This is the correction of issue #1.
Plus make the TTL Remover with gradle architecture to create an application.

### CODE REVIEW

**Comment by kgreav** [comment]
Hi @fasar, sorry for the delay. I'll have a look at this in more detail today/tomorrow. Thanks a lot for the contribution!

File: README.md [hunk 1]
```diff
@@ -8,55 +8,42 @@ This tool is implemented based on Cassandra 2.2.
...
-#### Remove TTL and create new SSTable
+4. You can use the application in with scripts in [PATH]/bin
 
-1. Change change folder to cassandra/tool/bin and the command for running the tool is as the following:
 
- `./TTLRemover [full path to the sstable folder] -p <output path>`
+#### Remove TTL and create new SSTable
 
- Note: your output path must end with `\`. Then, all the ttl-removed sstable is located in the `tools/bin/<output path>`.
+1. Change change folder to [PATH]/bin and the command for running the tool is as the following:
```
Comment (kgreav) [inline README.md]: I know it was there before, but seeing as we're here do you mind removing the repeated "change"?

File: gradlew [hunk 2]
```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
```
Comment (kgreav) [inline gradlew:1]: Mind chmod +x'ing this file?

File: build.gradle [hunk 3]
```diff
@@ -0,0 +1,32 @@
+group 'org.apache.cassandra'
+version '1.0'
+
+apply plugin: 'java'
+apply plugin: 'application'
+
+sourceCompatibility = 1.8
+
+repositories {
+    maven { url "http://172.16.20.36:8081/repository/maven-public/" }
```
Comment (kgreav) [inline build.gradle]: Can you remove this maven repo? You should be able to configure private repos in your maven settings.xml, and that will mean you don't need to include them in your build files.

**Comment by kgreav** [comment]
I've pushed a PR to your [branch](https://github.com/fasar/TTLRemover/pull/1) which addresses my comments and cleans up a bit of other stuff. If you can review/merge and test TTLRemoverKeyspace then we can pull here.

**Comment by fasar** [comment]
I update file as you wanted. 
I test it with your modificiation and I just change the way the script run the app.
Now it is needed CASSANDRA_HOME instead of CASSANDRA_INCLUDE.

CASSANDRA_INCLUDE was not able to define a good CLASSPATH when this script was executed in an other directory than cassandra.

**Comment by kgreav** [comment]
Thanks for all your efforts @fasar !

**Pull request merged**